R/runs.R

```diff
@@ -253,28 +246,17 @@ log_batch <- function(metrics = data.frame(), params = data.frame(), tags = data
   invisible()
 }
 
-has_nas <- function(df) {
-  any(is.na(df[, which(names(df) != "value")])) ||
-    any(is.na(df$value) & !is.nan(df$value))
-}
-
 validate_batch_input <- function(input_type, input_dataframe, expected_column_names) {
 
   if (is.null(input_dataframe) || nrow(input_dataframe) == 0) {
     return()
   } else if (!setequal(names(input_dataframe), expected_column_names)) {
     msg <- paste(input_type,
-      " batch input dataframe must contain exactly the following columns: ",
-      paste(expected_column_names, collapse = ", "),
-      ". Found: ",
-      paste(names(input_dataframe), collapse = ", "),
-      sep = ""
-    )
-    abort(msg)
-  } else if (has_nas(input_dataframe)) {
-    msg <- paste(input_type,
-      " batch input dataframe contains a missing ('NA') entry.",
-      sep = ""
+                 " batch input dataframe must contain exactly the following columns: ",
+                 paste(expected_column_names, collapse = ", "),
+                 ". Found: ",
+                 paste(names(input_dataframe), collapse = ", "),
+                 sep = ""
     )
```

Comment:
had to delete this so i can pass `NA` to `toJSON` in the `log_batch` call. they will be converted to `NULL`s there